### PR TITLE
Bring back the observer clock for D2k

### LINKED ISSUE #9461: D2K clock in replay viewer is gone in latest release
The clock from the previous release's (06-14) replay viewer is gone with the new (09-19).

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26985328-d2k-clock-in-replay-viewer-is-gone-in-latest-release?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by penev92**
Fixes #9461.

### CODE REVIEW

File: mods/d2k/chrome/ingame-observer.yaml [hunk 1]
```diff
@@ -18,6 +18,25 @@ Container@OBSERVER_WIDGETS:
 			Text: Statistics (F1)
 			Font: Bold
 			Key: f1
+		Container@GAME_TIMER_BLOCK:
+			Logic: GameTimerLogic
+			X: WINDOW_RIGHT/2 - WIDTH
```
Comment (obrakmann) [inline mods/d2k/chrome/ingame-observer.yaml]: `- WIDTH / 2`, otherwise it's not centered correctly.

**Comment by obrakmann** [comment]
:+1: after the comment has been addressed.

File: mods/d2k/chrome/ingame-observer.yaml (hunk 1 above)
Comment (pchote) [inline mods/d2k/chrome/ingame-observer.yaml]: `(WINDOW_RIGHT - WIDTH) / 2`

**Comment by penev92** [comment]
Centered. Not convinced it looks better though.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/e984d1b9595eb39827be47f27f9a92c24db7aa50...a30f9271206f2bf270ed5b822ae7f4750701ae1d)

**Pull request merged**